### DISCUSSION TITLE
Do Plural clusters scale-to-zero?

### DISCUSSION THREAD

**Discussion by ianyoung** [pluralsh/plural#957] [General]
In GKE 1.22 and later clusters can scale to zero. Would this still be the case when deploying to GKE using Plural?

**Comment by michaeljguarino** [comment]
On GCP we do simply provision a recent version of GKE so it will be entirely compatible, but it's worth noting that a lot of this depends on the application, eg:

* if it has a long-lived webserver or data tier, it likely will keep requesting capacity preventing an autoscaler from thinking it can scale to 0
* it likely also needs some form of KEDA autoscaler to trigger scale down if compatible or a user wants to opt in.

We were working on implementing the latter but it got deprioritized a bit.  It might be worth bringing back up though.
